data/ueber.json

```
@@ -1,22 +1,22 @@
 [
     {
         "title": "Verwaltungen",
-        "txt": "Bei Code for Germany engagieren sich Bürger&ast;innen, die Expertise im Umgang mit Offenen Daten haben und weiterentwickeln. Sie suchen den Austausch mit den Verwaltungen und stehen ihnen beratend zur digitalen Stadtentwicklung zur Seite. Verwaltungen auf allen Ebenen können Legitimation für ihr eigenes Handeln schaffen, indem sie die öffentlichen Daten ihrer Kommunen für Bürger*innen öffnen. Diese Transparenz ist auch ein guter Weg, um Bürger*innen von den eigenen Angeboten zu überzeugen."
-    },
-    {
-        "title": "Jetzt die Befreiung von Daten unterstützen!",
-        "txt": "* Offene Daten zur Verfügung stellen.\n* Fördergelder für die Erschließung Offener Daten bereitstellen.\n* Anwendungen aus Offenen Daten in die eigene Arbeit einbinden.\n* Eigene Expertise und Beratung teilen."
+        "txt": "Wir unterstützen Verwaltungen auf ihrem Weg, Souveränität über ihre Daten zu erlangen. Im Austausch mit der Open-Data-Community werden digitale Anwendungen und Datenstandards entwickelt, die den Austausch zwischen Behörden verbessern, Verwaltungswege vereinfachen und die Gesellschaft zur Teilhabe mobilisieren. Mit der Durchsetzung von Datenstandards und durch den Einsatz Freier und Offener Software werden Verwaltungen unabhängig von Großkonzernen und damit flexibler und handlungsfähiger. Die Umstellung auf Offene Daten schafft Transparenz gegenüber der Öffentlichkeit und Legitimation für das Handeln der Verwaltungen."
     },
     {
         "title": "Politik",
-        "txt": "Für Entscheidungsträger&ast;innen in der Politik sind Open Government Strategien von besonderer Bedeutung. Denn durch diese wird der Dialog zwischen Bürger&ast;innen und Politiker&ast;innen verstetigt und öffentliches Wissen auf eine breite Basis gestellt. Im Austausch mit der Code for Germany Community können sie mehr über die Probleme und digitalen Lösungsansätze in ihren Wahlkreisen erfahren und ihre Verantwortung gegenüber den Wähler*innen stärken."
+        "txt": "Entscheidungsträger\*innen in der Politik können sich in Gesetzgebungsprozessen für Transparenz, Offene Daten und Open-Government-Strategien einsetzen. Damit stärken sie demokratische Verfahren, Wissensbildung und Partizipation. Im Austausch mit unserer Community erfahren sie mehr über die digitalen Lösungsansätze für Herausforderungen in ihren Wahlkreisen und können den Austausch zwischen sich und der Zivilgesellschaft entsprechend suchen und fördern."
```

Ich glaube `&ast;` war aus einem guten Grund hier drin. Das ist der HTML-Code für `*`.
```suggestion
        "txt": "Entscheidungsträger&ast;innen in der Politik können sich in Gesetzgebungsprozessen für Transparenz, Offene Daten und Open-Government-Strategien einsetzen. Damit stärken sie demokratische Verfahren, Wissensbildung und Partizipation. Im Austausch mit unserer Community erfahren sie mehr über die digitalen Lösungsansätze für Herausforderungen in ihren Wahlkreisen und können den Austausch zwischen sich und der Zivilgesellschaft entsprechend suchen und fördern."
```